platform-includes/configuration/sample-rate/flutter.mdx

```diff
@@ -0,0 +1,9 @@
+```dart
+import 'package:flutter/widgets.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+Future<void> main() async {
+  // Capture only 25% of events
+  await SentryFlutter.init((options) => options.sampleRate = 0.25);
+}
+```
```

Comment:
do we need the import and add void main? at this point the user should've gone through the intro anyway, imo I'd prefer this and highlight the sampleRate line

```dart
await SentryFlutter.init(
  (options) {
		// Capture only 25% of events
    options.sampleRate = 0.25;
  },
);
```

wdyt